RInAppMessaging/Classes/Views/SlideUpView.swift

```diff
@@ -60,8 +60,8 @@ internal class SlideUpView: UIView, SlideUpViewType {
             return
         }
 
-        //swiftlint:disable:next todo
-        //TODO: (Daniel Tam) - Support TOP direction for slide-up
+        // swiftlint:disable:next todo
+        // TODO: (Daniel Tam) - Support TOP direction for slide-up
```

Comment:
I think if this is something we potentially want it should be added to the backlog rather than be a TODO in the code.

Reply:
It's a legacy code. I don't think we'll need that functionality in the nearest future. We don't even support left/right in the dashboard. I'll just remove those todos